content/events/2023-02-20-Freiburg-Workshop/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Workshop on High-Throughput Data Analysis with Galaxy
+date: '2023-02-20'
+days: 5
+continent: EU
+location:
+  name: "University of Freiburg, Germany"
+gtn: true
+contact: "Anika Erxleben"
+tags: ["workshop"]
+subsites: [all]
+---
+
+We are offering a Galaxy workshop on HTS Data analysis. This is a students workshop at the University of Freiburg where we have a few places left. If you are interested or need more information, please contact Anika Erxleben (erxleben@informatik.uni-freiburg.de).
```

Comment:
This is a students workshop at the University of Freiburg where we have a few places left for other Galaxy users as well (PhD students, PostDocs).